github.js

```diff
@@ -1,72 +1,64 @@
 /* Copyright (c) 2014-2015 Richard Rodger, MIT License */
```

Comment:
Looks epic, can we comment out this file ala, https://github.com/rjrodger/nodezoo-info/blob/code-change/info.js